x-pack/test/apm_api_integration/tests/alerts/rule_registry.ts

```diff
@@ -411,82 +410,6 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           }
         `);
 
-        const now = new Date().getTime();
-
-        const { body: topAlerts, status: topAlertStatus } = await supertest
-          .get(
-            format({
-              pathname: '/api/observability/rules/alerts/top',
```

Comment:
Not sure why this is part of the APM api integration tests, but that route doesn't exist anymore. The assertions via the ES api above should still provide enough value, though.